pkg/controller/controller.go

```diff
@@ -296,7 +296,7 @@ func (c *Controller) sync(domain string) error {
 func (c *Controller) zmsGetSignedDomains(domain string) (*zms.SignedDomains, bool, error) {
 	d := zms.DomainName(domain)
 	master := false
-	signedDomain, _, err := c.zmsClient.GetSignedDomains(d, "", "", &master, "")
+	signedDomain, _, err := c.zmsClient.GetSignedDomains(d, "", "", &master,&master, "")
```

Comment:
Please give more context/information regarding this change.

Reply:
Hi @zetaj ,
Please find the attached screenshot of the function `GetSignedDomains()`.
The `conditions` field has been added as a parameter to the updated package. So to accomodate this, we are sending it as a boolean set to `false`.

![Screenshot 2024-06-03 at 10 17 01 AM](https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/bc490ce9-e71b-4280-8433-8f9fdfca3771)